src/autocomplete/EmojiAutocomplete.js

```diff
@@ -8,7 +8,7 @@ import Popup from '../common/Popup';
 import EmojiRow from '../emoji/EmojiRow';
 import { getFilteredEmojis } from '../emoji/data';
 import { useSelector } from '../react-redux';
-import { getActiveImageEmojiByName } from '../selectors';
```

Comment:
It looks like this branch cuts all the call sites of `getActiveImageEmojiByName`, replacing them with the new selector. So it can delete that old selector, too.